main/busybox/busyboxconfig

```diff
@@ -901,6 +910,8 @@ CONFIG_NETSTAT=y
 CONFIG_FEATURE_NETSTAT_WIDE=y
 CONFIG_FEATURE_NETSTAT_PRG=y
 CONFIG_NSLOOKUP=y
+# CONFIG_FEATURE_NSLOOKUP_BIG is not set
```

Comment:
This option has the following effect:

> Use internal resolver code instead of libc

Do we want to enable it? I would personally prefer not to.